integrations.rst

```diff
@@ -20,6 +20,7 @@ Xcode.
    integrations/xcode
    integrations/meson
    integrations/android
+   integrations/artifactory_buildinfo
```

Comment:
Probably we want a ``jfrog`` section, it is likely that we will add further integration details (Xray) here.

Reply:
We thit but wouldn't i be confusing? I mean, we support conan package repositories in JFrog Artifactory by default, so we didn't want to create a wide "jfrog"/"artifactory" integration that could mislead users (especially new ones). This is why we used "build info" specifically.

Reply:
ok, no problem